### ISSUE TITLE
Error on Install of newest version

### ISSUE THREAD

**Issue by joelmayer** [MarshallOfSound/Google-Play-Music-Desktop-Player-UNOFFICIAL-#164]
I tried to updated, received error.  I uninstalled old version, tried to install new version, same error.

![image](https://cloud.githubusercontent.com/assets/16683920/12716569/3626cec6-c8ea-11e5-9aa9-ae700aaa0442.png)

Interestingly, the file version doesn't show as 2.1.0, but I downloaded it twice from the update notice from the app, and twice for github.
![image](https://cloud.githubusercontent.com/assets/16683920/12716596/587e251e-c8ea-11e5-8629-7fbd246afc7f.png)

**Comment by JoshuaGarrison27** [comment]
Can you post some of the Log file?

**Comment by joelmayer** [comment]
Yep, sorry, meant to do that.  Attached.
[Google_Play_Music_Desktop_Player_20160201135855.txt](https://github.com/MarshallOfSound/Google-Play-Music-Desktop-Player-UNOFFICIAL-/files/112174/Google_Play_Music_Desktop_Player_20160201135855.txt)

**Comment by JoshuaGarrison27** [comment]
Forgive me for asking basic questions but i have to ask based on what I am seeing in the log.

1) Do you have admin rights on your machine? Did you try to "Run as administrator"?
2) Have you tried a reboot? (Just in case another process on the computer is halting this for some reason)

**Comment by joelmayer** [comment]
I had rebooted, but not "Run as Administrator".  Did that.  

I only had an option to uninstall through installer.  Then, I ran again and it installed, but I am thinking that it's not the right version, because on application launch, it prompted me again to update.  Went through the whole process of download, uninstall, reinstall.  

Still prompts me to update.

![image](https://cloud.githubusercontent.com/assets/16683920/12717282/8a84f718-c8ef-11e5-8a32-1d21fd6e3d6f.png)
